NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -351,6 +351,21 @@ private void OnVehiclePrefabLoaded(TechType techType, GameObject prefab, NitroxI
 
             VehicleChildObjectIdentifierHelper.SetInteractiveChildrenIds(gameObject, interactiveChildIdentifiers);
 
+            // TODO: Remove this when merging
```

Comment:
this is for testing purpose

Reply:
This should be in PR description, no need to include this inside the code with directives to remove afterwards